firestore-send-email/functions/src/index.ts

```diff
@@ -73,6 +73,7 @@ interface QueuePayload {
   bccUids?: string[];
   from?: string;
   replyTo?: string;
+  headers?: any;
```

Comment:
FYI: Taken from [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/nodemailer/v3/index.d.ts#L157). Headers are flexible so `any` works in this case.

Reply:
Since the postinstall says to provide an array or object, should we enforce that here?

Reply:
Pls see mbleigh@'s comment below.  Is this comment thread here still applicable?

Reply:
This is only for development, it won't have impact on the user experience. As NodeMailer accepts `any` I replicated it here. That being said, I'll change the docs to accept just a key/value object - I don't think this type should change?

Reply:
@mbleigh For confirmation pls - thx!